recaptcha_enterprise/demosite/app/static/demo-6df0841a.js

```diff
@@ -4185,10 +4187,17 @@ class RecaptchaDemo extends s {
         `;
         break;
       case "Bad":
-        card = x`
-          <p>reCAPTCHA is ${percentage || "???"}% confident you're not bad.</p>
-          <img alt="Bad" src="${badbad}" />
-        `;
+        if (this.reason) {
+          card = x`
+            <p>Suspicious request. Reason: "${this.reason}".</p>
+            <img alt="Bad" src="${badbad}" />
+          `;
+        } else {
+          card = x`
+            <p>reCAPTCHA is ${percentage || "???"}% confident you're not bad.</p>
```

Comment:
Do we need to update this to support a score of 0 ?

Reply:
Actually, I have removed the else case. Because all BAD actions will have an associated reason with it.
Hence, the else case will never get executed.

Kindly review the error message if this is how it should be worded.
`Suspicious request. Reason: `